examples/drivers/windows/otLwf/command.c

```diff
@@ -119,11 +121,28 @@ otLwfCmdInitialize(
         if (pFilter->cmdInitTryCount >= 10)
         {
 #else
-        NtStatus = otLwfCmdGetProp(pFilter, NULL, SPINEL_PROP_INTERFACE_TYPE, SPINEL_DATATYPE_UINT_PACKED_S, &InterfaceType);
+        NtStatus = otLwfCmdGetProp(pFilter, NULL, SPINEL_PROP_PROTOCOL_VERSION, "ii", &MajorVersion, &MinorVersion);
         if (!NT_SUCCESS(NtStatus))
         {
 #endif
             Status = NDIS_STATUS_NOT_SUPPORTED;
+            LogError(DRIVER_DEFAULT, "Failed to query SPINEL_PROP_PROTOCOL_VERSION, %!STATUS!", NtStatus);
+            break;
+        }
+        if (MajorVersion != SPINEL_PROTOCOL_VERSION_THREAD_MAJOR ||
+            MinorVersion != SPINEL_PROTOCOL_VERSION_THREAD_MINOR)
```

Comment:
Mind adding a special one-time case to black-list the previous version (could be removed at the next major version increment)?

Doing a major version rev right now would cause all sorts of chaos on our end.